README.md

```diff
@@ -149,18 +183,27 @@ For example if the `Content-Type` is `application/json` then the request body ma
 }
 ```
 
-Note: both query and mutation operations can be sent over POST requests.
-
 # Response
 
-When a GraphQL server receives a request, it must return a well‐formed response. The server’s
+When a GraphQL server receives a request, it must return a well‐formed response. The server's
 response describes the result of executing the requested operation if successful, and describes
 any errors encountered during the request.
 
 ## Body
 
 If the server's response contains a body it should follow the requirements for [GraphQL response](https://graphql.github.io/graphql-spec/June2018/#sec-Response).
 
+A server MUST return a `Content-Type` HTTP Header with a value of a valid GraphQL content type. By default the response MUST be serialized as JSON and MUST include a  "Content-type: application/graphql+json` header.
+
+If another content type is preferable to a client, it MAY include an `Accept` HTTP header listing other acceptable content types in order of preference. In this case a client SHOULD include `application/graphql+json` in the list, according to their preferred priority.
+
+If no `Accept` header is given, the server MUST respond with a content type of `application/graphql+json`.
```

Comment:
Perhaps weaken this statement and add non-normative note suggesting that this default be application/graphql+json to support legacy clients, to allow more wider implementations.